public/content/lessons/2017/implicit-differentiation/index.mdx

```diff
@@ -2,9 +2,278 @@
+
+## Related Rates
+
+Imagine a $5$ meter long ladder held up against a wall, where the top of the ladder starts of $4$ meters above the ground, which, by the pythagorean theorem, means the bottom is $3$ meters away from the wall.
+
+<Figure
+  image="figures/ladder-setup.png" 
+/>
+
+And say it's slipping down the wall in such a way that the top of the ladder is dropping at $1$ meter per second. The question is, in that initial moment, what is the rate at which the bottom of the ladder is moving away from the wall.
+
+<Figure
+  image="figures/ladder-rate-of-change.png"
+  video="figures/ladder-rate-of-change.mp4"
+  show="video"
+/>
+
+It's interesting, right? That distance from the bottom of the ladder to the wall is $100\%$ determined by the distance between the top of the ladder and the floor, so we should have enough information to figure out how the rates of change for each value depend on each other, but it might not be entirely clear how exactly you relate those two.
+
+First thing's first, it's always nice to give names to the quantities we care about. So label the distance from the top of the ladder to the ground $y(t)$, written as a function of time because it's changing. Likewise, label the distance between the bottom of the ladder and the wall $x(t)$. They key equation that relates these terms is the pythagorean theorem: $x(t)^2 + y(t)^2 = 5^2$. What makes this equation powerful is that it's true at all points in time.
+
+<Figure
+  image="figures/ladder-time-variable.png" 
+/>
+
+One way to solve this would be to isolate $x(t)$, figure out what what $y(t)$ must be based this $1$ meter/second drop rate, then take a derivative of the resulting function; $dx/dt$, the rate at which $x$ is changing with respect to time.
+
+<Figure
+  image="figures/ladder-traditional-derivative.png" 
+/>
+
+And that's fine; it involves a couple layers of using the chain rule, and it will definitely work for you. But I want to show a different way to think about the same problem.
+
+This left-hand side of the equation is a function of time, right? Which just so happens to equal a constant, meaning this value evidently doesn't change while time passes. 
+
+<Figure
+  image="figures/ladder-function-of-time.png" 
+/>
+
+However, this expression is dependent on time which we can manipulate like any other function with $t$ as an input. In particular, we can take a derivative of the left hand side, which is a way of saying "If I let a little bit of time pass, $dt$, which causes $y$ to slightly decrease, and $x$ to slightly increase, how much does this expression change".
+
+On the one hand, we know that derivative should be $0$, since this expression is a constant, and constants don't care about your tiny nudge to time, they remain unchanged. But on the other hand, what do you get when you computer the derivative of this function of time?
```

Comment:
```suggestion
On the one hand, we know that derivative should be $0$, since this expression is a constant, and constants don't care about your tiny nudge to time; they remain unchanged. But on the other hand, what do you get when you compute the derivative of this as a function of time?
```